app/Policies/GamePolicy.php

```diff
@@ -96,7 +96,7 @@ public function updateField(User $user, Game $game, string $fieldName): bool
             Role::DEVELOPER_JUNIOR => ['GuideURL', 'Developer', 'Publisher', 'Genre', 'released_at', 'released_at_granularity'],
 
             Role::DEVELOPER => ['Title', 'GuideURL', 'Developer', 'Publisher', 'Genre', 'released_at', 'released_at_granularity'],
-            Role::DEVELOPER_STAFF => ['Title', 'GuideURL', 'Developer', 'Publisher', 'Genre', 'released_at', 'released_at_granularity'],
+            Role::DEVELOPER_STAFF => ['Title', 'sort_title', 'GuideURL', 'Developer', 'Publisher', 'Genre', 'released_at', 'released_at_granularity'],
```

Comment:
I would imagine `DEVELOPER_STAFF` would encompass members of DevC and QAM at some point in the future. It makes sense they would also be authorized to override this field with a custom value.